src/test/java/com/softserveinc/dokazovi/controller/VersionControllerTest.java

```diff
@@ -16,14 +17,11 @@
 
 	@Autowired
 	private MockMvc mockMvc;
-	@Autowired
-	private BuildVersion buildVersion;
 
 	@Test
-	void shouldReturnBuildVersion() throws Exception {
-		String version = String.format("{\"version\":\"%s\"}", buildVersion.getVersion());
+	void getBuildVersion_Get_isOk() throws Exception {
 		this.mockMvc.perform(MockMvcRequestBuilders.get("/version"))
```

Comment:
Please use EndPoint constants instead of "/version"

Reply:
done